database/migrations/2017_06_25_200318_add_contact_avatar_location.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddContactAvatarLocation extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('contacts', function (Blueprint $table) {
+            $table->enum('avatar_location', ['s3', 'local'])->after('avatar_file_name')->default('local');
```

Comment:
Enum will break postgresql migrations. We need to avoid them. Can you rewrite the migration without them, and perhaps simple add `$table->string` type?